README.md

```diff
@@ -107,42 +103,39 @@ our [code of conduct](CODE_OF_CONDUCT.md) in all interactions both on and offlin
 
 If you want to report a problem or suggest an enhancement we'd love for you to
 [open an issue](../../issues) at this github repository because then we can get right on
-it. But you can also reach us on the [NeuroTechX slack #moabb
-channel][link_neurotechx_signup] where we are happy to help!
-
-## Find out more
-
-You might be interested in:
+it.
 
-- [MOABB documentation][link_moabb_docs]
-
-And of course, you'll want to know our:
-
-- [Contributors' guidelines](CONTRIBUTING.md)
-- [Roadmap](ROADMAP.md)
+For a less formal discussion or exchanging ideas, you can also reach us on the [Gitter
+channel][link_gitter] or join our weekly office hours! This an open video meeting
+happening every Thursday at [18:30 GMT+1](https://time.is/en/Paris), please ask the link
+on the gitter channel. We are also on [NeuroTechX slack #moabb
+channel][link_neurotechx_signup].
 
 ## Thank you
```

Comment:
Also do we need this sectiion?

I mean thanking is written in the very beginning of this document and here it's just repeated.
For me this file is more about technology than beautiful words. =)